### PR TITLE
fix(signet): #MED-351 fix signet problems

### PR DESCRIPTION
**Pull request by Mattox0**
## Describe your changes

Fix some signet problems

## Checklist tests

- Check if signet page works

## Issue ticket number and link

[MED-351](https://jira.support-ent.fr/browse/MED-351)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/modals/signet-property/SignetProperty.tsx [hunk 1]
```diff
@@ -118,7 +117,8 @@ export const SignetProperty: React.FC<SignetPropertyProps> = ({
         notify(t("mediacentre.modal.signet.ask.image"), "danger");
         return;
       }
-      const payload: SignetUpdatePayload = {
+      const payload: any = {
```
Comment (ThomasSevagen) [inline frontend/src/components/modals/signet-property/SignetProperty.tsx:120]: obligé de mettre un any?

**Review by alicedraillard — APPROVED** [review]

File: frontend/src/components/modals/signet-publish/SignetPublish.tsx [hunk 2]
```diff
@@ -56,6 +59,11 @@ export const SignetPublish: React.FC<SignetPublishProps> = ({
         plain_text: convertKeyWords(toasterResources[0].plain_text).map(
           (keyword) => ({ label: keyword }),
         ),
+        published:
+          !!publicSignets?.find(
```
Comment (ThomasSevagen) [inline frontend/src/components/modals/signet-publish/SignetPublish.tsx:63]: tu devrai utiliser le .some() tu aurai directement un booleen

Comment (ThomasSevagen) [reply frontend/src/components/modals/signet-publish/SignetPublish.tsx:63]: .some() au moins un row de l'array remplis ta condition = true

File: frontend/src/components/modals/signet-property/SignetProperty.tsx (hunk 1 above)
Comment (Mattox0) [reply frontend/src/components/modals/signet-property/SignetProperty.tsx:120]: malheuresement oui, car on reprends les données qu'on reçoit (on reçoit pas les mêmes à chaque fois)

**Review by ThomasSevagen — APPROVED** [review]

**Pull request merged**